lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -260,6 +262,25 @@ void LocalEnforcer::aggregate_records(
   }
   complete_termination_for_released_sessions(
       session_map, sessions_with_reporting_flows, session_update);
+  cleanup_dead_sessions(dead_sessions_to_cleanup);
+}
+
+void LocalEnforcer::cleanup_dead_sessions(
+    const ImsiAndIPSet dead_sessions_to_cleanup) {
+  if (dead_sessions_to_cleanup.size() == 0) {
+    return;
+  }
+  MLOG(MINFO) << "Deactivating" << dead_sessions_to_cleanup.size()
+              << " flows that are attached to non-existent sessions";
+  Teids empty_teids;
+  for (const ImsiAndIPs& info : dead_sessions_to_cleanup) {
+    pipelined_client_->deactivate_flows_for_rules_for_termination(
+        info.imsi, info.ipv4, info.ipv6, empty_teids, {}, {},
+        RequestOriginType::GX);
+    pipelined_client_->deactivate_flows_for_rules_for_termination(
+        info.imsi, info.ipv4, info.ipv6, empty_teids, {}, {},
+        RequestOriginType::GY);
+  }
```

Comment:
is it possible that just sending one request with an empty `RequestOriginType` will remove both?

Reply:
@koolzz ? We could define a new origin type : ANY or something like that.

Reply:
sure lets define ANY

Reply:
@koolzz Added a WILDCARD enum to specify both. Added the corresponding change on the PipelineD receiving side.